src/openvtb/conf/api.py

```diff
@@ -13,5 +13,8 @@
 REST_FRAMEWORK["DEFAULT_PAGINATION_CLASS"] = (
     "vng_api_common.pagination.DynamicPageSizePagination"
 )
+REST_FRAMEWORK["DEFAULT_AUTHENTICATION_CLASSES"] = (
+    "rest_framework.authentication.TokenAuthentication",
```

Comment:
I remember some time ago there was an issue with this token model using the actual token as the PK (https://github.com/maykinmedia/objects-api/issues/445), but I guess that's resolved now? `/admin/authtoken/tokenproxy/1/change/`